config/locales/de.yml

```diff
@@ -358,6 +358,7 @@ de:
     blog:
       description: 'Wenn du für uns schreiben möchtest, stelle dich gerne per E-Mail bei uns vor: %{email}! Auch anonyme Einsendungen werden berücksichtigt.'
     contribute:
+      contributor_avatar_alt: "Mitwirkende Biographie für %{name}"
```

Comment:
Just to be consistent with our linting, let's use single quotes instead of double quotes.